palm/plugins/dbt/commands/cmd_snapshot.py

```diff
@@ -10,15 +10,15 @@
 def cli(ctx, 
         persist: bool,
         fast: bool,
-        select: Optional[tuple]=()):
+        select: Optional[tuple] = tuple()):
     """ Executes the DBT snapshots."""
 
     if fast:
         cmd = "dbt snapshot"
     else:
         cmd = "dbt clean && dbt deps && dbt snapshot"
     if select:
-        cmd += f" --select {select}"
+        cmd += f" --select " + " ".join(select)
```

Comment:
you can drop this all in the f-string if you'd like `f" --select {' '.join(select) }"`

Reply:
hm, neat. Future improvement 🚀